src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1349,4 +1349,54 @@ private static int indexOfSameAllele(final VariantContext vc, final Allele allel
 
         return -1;
     }
+
+    /**
+     * Get the number of called alleles for a genotype. Also, increment the number of called alternate alleles.
+     *
+     * @param calledAltAlleles number of called alternate alleles for all genotypes
+     * @param genotype         genotype
+     * @return incremented called alleles
+     * @throws IllegalArgumentException if calledAltAlleles or genotype are null
+     */
+    public static int getCalledChromosomeCounts(final Map<Allele, Integer> calledAltAlleles, final Genotype genotype) {
```

Comment:
It's weird that this method is a non-void method that updates it's input.  Since computing the calledAlleles number is essentially orthogonal to updating could you split the functionality?  It seems like these are packaged together in order to reduce complexity to the caller since they're needed in multiple places, but I think that could be avoided by refactoring so that it's only needed by a single function that can be called in both places.